azure.yaml

```diff
@@ -1,3 +1,18 @@
+- active: false
+  authorities:
+  - name: app-operator
+    version: 1.0.0
+  - name: azure-operator
+    version: 2.8.0
+  - name: cert-operator
+    version: 0.1.0
+  - name: chart-operator
+    version: 0.7.0
+  - name: cluster-operator
+    provider: azure
+    version: 0.22.0-dev
+  date: 2019-01-29T13:00:00Z
+  version: 11.1.0
```

Comment:
```suggestion
  version: 11.0.1
```

Reply:
We introduced a new feature of disabling TCP DNS checks. Is this is a bug fix?

Reply:
Is it a feature or is it a bugfix? Seemed like a bug affecting a customer to me. Ignore me if wrong.

Reply:
Not a bug, Customer dosent support TCP DNS. The tool checked per default. we allow to disable this check with this release. So for me it's a feature